drivers/xen/events.c

```diff
@@ -60,7 +68,22 @@ int unbind_event_channel(evtchn_port_t port)
 		__func__, port);
 
 	event_channels[port].cb = empty_callback;
-	event_channels[port].priv = NULL;
+	event_channels[port].priv = &event_channels[port];
+	events_missed[port] = false;
+
+	return 0;
+}
+
+int get_missed_events(evtchn_port_t port)
+{
+	__ASSERT(port < EVTCHN_2L_NR_CHANNELS,
```

Comment:
I wonder if it is better to give a warning instead of an assert? Is it a fatal error?

Reply:
I think it is, I've also followed the overall way other functions are doing to handle the port argument

Reply:
OK